components/orbit-controls/lib/OrbitControls.js

```diff
@@ -1,1041 +1,1287 @@
-/**
- * @author qiao / https://github.com/qiao
- * @author mrdoob / http://mrdoob.com
- * @author alteredq / http://alteredqualia.com/
- * @author WestLangley / http://github.com/WestLangley
- * @author erich666 / http://erichaines.com
- */
-
-// This set of controls performs orbiting, dollying (zooming), and panning.
+/* THREE.OrbitControls code comes from:
+https://github.com/mrdoob/three.js/blob/master/examples/jsm/controls/OrbitControls.js
```

Comment:
Can you please specify the exact commit and three release
https://github.com/mrdoob/three.js/blob/0fbae6f682f6e13dd9eb8acde02e4f50c0b73935/examples/jsm/controls/OrbitControls.js three r151

For @dmarcos the file is a copy of the above commit with just this
```js
import {
	EventDispatcher,
	MOUSE,
	Quaternion,
	Spherical,
	TOUCH,
	Vector2,
	Vector3
} from 'three';
```

replaced by

```js
const	{
  EventDispatcher,
	MOUSE,
	Quaternion,
	Spherical,
	TOUCH,
	Vector2,
	Vector3
} = THREE;
```
to avoid duplicate threejs warning.

Note: Another way to update this could be to update three and webpack packages, properly configure webpack.config.js to use three external, remove `lib/OrbitControls.js` from the repo and just using:
`import { OrbitControls } from "three/examples/jsm/controls/OrbitControls.js";`